spyder/plugins/projects/plugin.py

```diff
@@ -213,18 +215,19 @@ def register(self):
                 lambda v: outline_explorer.update_all_editors())
         if self.main:
             self.sig_project_closed.connect(
-                lambda v: self.main.set_window_title())
+                lambda v, s: self.main.set_window_title())
         if self.completions:
             self.sig_project_closed.connect(
                 functools.partial(self.completions.project_path_update,
                                   update_kind=WorkspaceUpdateKind.DELETION,
                                   instance=self))
         if self.editor:
             self.sig_project_closed.connect(
-                lambda v: self.editor.setup_open_files())
+                lambda v, s: (None if s
+                              else self.editor.setup_open_files()))

```

Comment:
Please revert. Now only `self.sig_project_closed[bool]` is connected to this slot.